test/functional/http/get/multisignatures.js

```diff
@@ -118,7 +119,8 @@ describe('POST signatures/sign (regular account)', function () {
 				node.expect(res.body.transaction).to.have.property('id').to.equal(transaction.id);
 				var signature = node.lisk.multisignature.signTransaction(transaction, multisigAccount.password);
 				sendSignature(signature, transaction, function (err, res) {
-					node.expect(err).not.to.be.empty;
+					node.expect(res).to.have.property('success').to.not.be.ok;
+					node.expect(res).to.have.property('message').to.equal('Error processing signature: Transaction not found');
```

Comment:
Description, why `should be impossible to sign the transaction`?

Reply:
We need to rewrite these tests. They are not enough. The purpose of this PR is not that.